pkg/kubeapiserver/authenticator/patch_authenticator.go

```diff
@@ -1,7 +1,6 @@
 package authenticator
 
 import (
-	"fmt"
```

Comment:
is this a revert of another patch? Then squash.

Reply:
Done.